source/common/res/features/hide-help/main.js

```diff
@@ -3,9 +3,19 @@
   if (typeof ynabToolKit !== 'undefined' && ynabToolKit.pageReady === true) {
     ynabToolKit.hideHelp = (function () {
       return {
+        hideHelp: 'true',
+
         invoke() {
-          // hide button by default
-          $('#hs-beacon').hide();
+          let hideHelp = ynabToolKit.shared.getToolkitStorageKey('hide-help');
+
+          if (typeof hideHelp === 'undefined' || hideHelp === null) {
```

Comment:
hide help shouldn't ever be undefined here...it will either be `true`/`false`/`null`

`if (hideHelp)` should suffice